generics/helpers/utils.js

```diff
@@ -539,36 +558,35 @@ function checkIfValidUUID(value) {
  */
 
 function md5Hash(value) {
-  return md5(value);
+	return md5(value)
 }
 
-
-
 module.exports = {
-  camelCaseToTitleCase : camelCaseToTitleCase,
-  lowerCase : lowerCase,
-  checkIfStringIsUrl : checkIfStringIsUrl,
-  hyphenCaseToCamelCase : hyphenCaseToCamelCase,
-  valueParser : valueParser,
-  convertStringToBoolean : convertStringToBoolean,
-  getAllBooleanDataFromModels : getAllBooleanDataFromModels,
-  epochTime : epochTime,
-  isValidMongoId : isValidMongoId,
-  convertProjectStatus : convertProjectStatus,
-  revertProjectStatus:revertProjectStatus,
-  revertStatusorNot:revertStatusorNot,
-  checkValidUUID : checkValidUUID,
-  createComparableDates : createComparableDates,
-  noOfElementsInArray : noOfElementsInArray,
-  operatorValidation : operatorValidation,
-  generateUniqueId : generateUniqueId,
-  getEndDate : getEndDate,
-  getStartDate : getStartDate,
-  convertStringToObjectId : convertStringToObjectId,
-  isValidMongoId : isValidMongoId,
-  filterLocationIdandCode : filterLocationIdandCode,
-  generateTelemetryEventSkeletonStructure : generateTelemetryEventSkeletonStructure,
-  generateTelemetryEvent : generateTelemetryEvent,
-  checkIfValidUUID : checkIfValidUUID,
-  md5Hash : md5Hash
-};
+	camelCaseToTitleCase: camelCaseToTitleCase,
+	lowerCase: lowerCase,
+	checkIfStringIsUrl: checkIfStringIsUrl,
+	hyphenCaseToCamelCase: hyphenCaseToCamelCase,
+	valueParser: valueParser,
+	convertStringToBoolean: convertStringToBoolean,
+	getAllBooleanDataFromModels: getAllBooleanDataFromModels,
+	epochTime: epochTime,
+	getGotenbergConnection: getGotenbergConnection,
```

Comment:
@MallanagoudaB Please move this function to a new file and call it there.path :  generics/services/gotenberg.js